components/sup/src/manager/service/hooks.rs

```diff
@@ -722,6 +722,8 @@ impl HookOutput {
         }
     }
 
+    // TODO fn: this method appears to not be used in the codebase, is it necessary?
```

Comment:
I'd be inclined to just rip this out if no one can think where this might be used.